### PR TITLE
feature: Add support for PyTorch 1.2

### PR DESCRIPTION
**Pull request by mvsusp**
*Description of changes:*
- add support for 1.2 PyTorch Containers

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

- [X] I have read the [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md) doc
- [X] I used the commit message format described in [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md#committing-your-change)
- [X] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [X] I have updated any necessary [documentation](https://github.com/aws/sagemaker-python-sdk/blob/master/README.rst) (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/sagemaker/fw_utils.py [hunk 1]
```diff
@@ -140,6 +149,18 @@ def _is_tf_14_or_later(framework, framework_version):
     )
 
 
+def _is_pt_12_or_later(framework, framework_version):
+    """
+    Args:
+        framework: Name of the frameowork
+        framework_version: framework version
+    """
+    asimov_lowest_pt = [1, 12, 0]
```
Comment (ChoiByungWook) [inline src/sagemaker/fw_utils.py]: Should gov region be handled here? We can do this in a separate PR.

Comment (laurenyu) [reply src/sagemaker/fw_utils.py]: if the map above is filled out, it should work fine. not sure why an extra method is needed...

Comment (ChoiByungWook) [reply src/sagemaker/fw_utils.py]: lauren pointed this out, but this should be [1,2,0] and not [1,12,0]

**Comment by ChoiByungWook** [comment]
Closing in favor of: https://github.com/aws/sagemaker-python-sdk/pull/1091

Error was due to not using the repack_model attribute of the model class when defining the container definition.
